docs/digger/admin/using-http-proxy.asciidoc

```diff
@@ -2,25 +2,26 @@
 
 When you configure the Digger Installer to use proxy, it will automatically reconfigure the following components:
 
-
 * The Jenkins server
 * The Android SDK pod used to download Android SDKs
-* Container/node that will run the build jobs, by adding the appropriate environment variables like `HTTP_PROXY` and `HTTPS_PROXY` in the inventory file - sample provided https://github.com/aerogear/digger-installer/blob/master/inventory-sample[here].
-* Ansible to install various tools used to provision the macOS servers(??? verify if this is correct or it should be "Ansible to install various macOS tools"
+* Container/node that will run the build jobs, by adding the appropriate environment variables like `HTTP_PROXY` and `HTTPS_PROXY` in the inventory file. 
+Sample provided in https://github.com/aerogear/digger-installer/blob/master/inventory-sample[AeroGear Digger GitHub repo].
+* Ansible to install various tools used to provision the macOS servers
+//(??? verify if this is correct or it should be "Ansible to install various macOS tools")
 
-These changes should allow the default build tools like Gradle and XCode to work with the HTTP proxy. (should or will?)
+These changes allow the default build tools (Gradle, XCode) to use HTTP proxy.
 
 === Configuration Options
 
 You can run AeroGear Digger behind a HTTP proxy server by defining the following Ansible Variables:
 
 |===
 | Variable name | Description
-
 | proxy_url
 | This URL is pasted into `http_proxy` and `https_proxy` environment variables.
 | proxy_host
-| The host name of the proxy server. It should not contain protocol(??? HTTP etc? Protocol definition?)
+| The host name of the proxy server. It should not contain protocol
```

Comment:
(??? HTTP etc? Protocol definition?)

Reply:
i.e. (http/https) The url should be <host>:<port> not <protocol>//:<host>:<port>

Reply:
would an example help here?

The host name of the proxy server, for example myproxy.example.com

Reply:
@laurafitzgerald could you please post an example? It is not clear to me from your example.

Reply:
sorry @pmacko1 The example lost some parts in md. Ill fix it now

Reply:
@pmacko1 updated now see above example now.

Reply:
resolved